### PR TITLE
Criminal

### PR DESCRIPTION
**Pull request by KarenMaza**
# Your checklist for this pull request

## About your game

What is your game about? 
Pushing a box.

How to play your game?
 Press WASD to move and push the jail cell on the yellow blob person.
## Code
Check off the items that are true.
- [x] The game was made using the [Sprig editor](https://editor.sprig.hackclub.com/).
- [x] The game is placed in the in the [`/games` directory](https://github.com/hackclub/sprig/tree/main/games).
- [x] The code is significantly different from all other games in the [Sprig gallery](https://sprig.hackclub.com/gallery) (except for games labeled "demo").
- [x] The game runs without errors. 
- [x] The name of the file/game contains only alphanumeric characters, `-`s, or `_`s.
- [x] The game name is not the same as the others from [gallery](https://sprig.hackclub.com/gallery)

## Image (If an image is used)

- [x] The image is in the [`/games/img` directory](https://github.com/hackclub/sprig/tree/main/games/img).
- [x] The name of the image matches the name of your file. Example: `sokoban_plus.js` -> `sokoban_plus.png`.

> Thanks for PR!

### CODE REVIEW

File: games/Criminal.js [hunk 1]
```diff
@@ -0,0 +1,260 @@
+/*
+/*
+@title: Criminal
+@author: Karen
+*/
+Instructions:
```
Comment (SamDev-7) [inline games/Criminal.js:6]:
```suggestion
/*
Instructions
```
Uh oh! There seems to be some issues with the comments, make sure they are properly commented.
Let us know if you need any help!

File: games/Criminal.js [hunk 2]
```diff
@@ -0,0 +1,260 @@
+/*
```
Comment (SamDev-7) [inline games/Criminal.js:1]:
```suggestion
```

**Review by SamDev-7 — CHANGES_REQUESTED** [review]
Thanks for the PR! I loved the art! Just some fixes with the comments and it'll be ready to go!

**Comment by LucasHT22** [comment]
good sprites! my suggestion is to add a background:)

**Comment by SamDev-7** [comment]
@KarenMaza You're soo close to getting your Sprig, don't give up!

**Comment by KarenMaza** [comment]
I'm not sure. I just used it because I made the box a jail cell and the other sprite is kind of hiding from the player. So the sprite is like a criminal running from the detective?

**Comment by LucasHT22** [comment]
oh, good idea!

**Comment by LucasHT22** [comment]
my suggestion is to add a background:)

**Review by SamDev-7 — APPROVED** [review]

**Comment by leomcelroy** [comment]
It's been a little while so I'm going to close this PR for now. 

Feel free to reopen if you want to continue working on your game!

**Pull request closed without merging**